src/components/common/Repository.tsx

```diff
@@ -0,0 +1,47 @@
+import { FiBook, FiCircle, FiStar } from "react-icons/fi";
+import { GoRepoForked } from "react-icons/go";
+import "../../styles/repository.css";
+
+type RepositoryProps = {
```

Comment:
```suggestion
type Props = {
```
I noticed that you are already defined a pattern of having the name of the component props as `Props`. So it would be nice to keep following your pattern in here.